<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainMenuState</class>
    <widget class="QWidget" name="MainMenuState">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1223</width>
                <height>680</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5">
                    <item>
                        <widget class="QLabel" name="label">
                            <property name="styleSheet">
                                <string notr="true">font: 36pt &quot;微软雅黑&quot;;</string>
                            </property>
                            <property name="text">
                                <string>Gwent</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <layout class="QGridLayout" name="gridLayout_3">
                            <item row="1" column="2">
                                <widget class="QPushButton" name="pushButton_6">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Exit</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QPushButton" name="pushButton_5">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Recover Game</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QPushButton" name="pushButton_4">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Card Group Edit</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QPushButton" name="pushButton_2">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Log In / Sign Up</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="2">
                                <widget class="QPushButton" name="pushButton_3">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Game Settings</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="0">
                                <widget class="QPushButton" name="pushButton">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">font: 24pt &quot;微软雅黑&quot;;</string>
                                    </property>
                                    <property name="text">
                                        <string>Start Game</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
